db/migrate/20200516021201_add_theme_to_users.rb

```diff
@@ -0,0 +1,6 @@
+class AddThemeToUsers < ActiveRecord::Migration[5.2]
+  def change
+    add_column :users, :theme, :string
+    change_column_default :users, :theme, "default"
```

Comment:
Should use from to, else the change is not reversible
https://guides.rubyonrails.org/active_record_migrations.html#changing-columns